src/Table/table.styles.js

```diff
@@ -0,0 +1,71 @@
+import ReactTable from "react-table";
+import styled from "styled-components";
+import "react-table/react-table.css";
+import colors from "../Colors";
+
+const WithoutConflictingDefaultStyles = styled(ReactTable)`
```

Comment:
cant `WithoutConflictingDefaultStyles` and `StyledTable` be merged into one const? Would that make sense to you? 
Just for me reading the code dont see the reasoning behind splitting it.

Reply:
Ah ok. I split them so there is a clear distinction between which styles we are stripping from react-table, and which ones we are adding as extras. 

If that still doesn't make sense, I can remove it?

Reply:
Makes sense. It is best this way now that you explained. Maybe add comment?

Reply:
Good idea :)